docs/interpreter_checklist.md

```diff
@@ -0,0 +1,67 @@
+# StableHLO Interpreter Checklist
+
+In this document, we summarize the guidelines while implementing and reviewing
+interpreter for an op. We have intentionally tied a few auxiliary action items,
+related to verifier and type inference, with the idea of making progress in
+those fronts alongside the interpreter implementation.
+
+1. While implementing the interpreter for an op, double check with related
+   implementations (e.g.
+   [hlo_interpreter](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/hlo/evaluator)
+   and
+   [mlir_interpreter](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/compiler/xla/mlir_hlo/tools/mlir_interpreter)).
+   This action item will allow mutual improvements.
+1. Make sure that the ODS follows standard [summary](https://github.com/openxla/stablehlo/issues/611).
```

Comment:
What is the standard summary? Let's write this down in the ticket.

Reply:
updated the ticket.